ui/src/shared/constants/fluxFunctions.ts

```diff
@@ -250,6 +250,28 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
     link:
       'https://v2.docs.influxdata.com/v2.0/reference/flux/functions/built-in/inputs/buckets/',
   },
+  {
+    name: 'chandeMomentumOscillator',
+    args: [
+      {
+        name: 'n',
+        desc: 'The period or number of points to use in the calculation.',
+        type: 'Integer',
+      },
+      {
+        name: 'columns',
+        desc: 'Columns to operate on. Defaults to `["_value"]`.',
+        type: 'Array of Strings`',
+      },
+    ],
+    package: '',
+    desc:
+      'Applies the technical momentum indicator developed by Tushar Chande.',
+    example: 'chandeMomentumOscillator(n: 10, columns: ["_value"])',
```

Comment:
Remove the `columns` param from this example to keep it simple. It's rare that a user will want to using something other than the default value here.

```
chandeMomentumOscillator(n: 10)
```